docs/README.md

```diff
@@ -0,0 +1,26 @@
+[![Build Status](https://api.travis-ci.org/daostack/arc.js.svg?branch=master)](https://travis-ci.org/daostack/arc.js)
+[![NPM Package](https://img.shields.io/npm/v/@daostack/arc.js.svg?style=flat-square)](https://www.npmjs.org/package/@daostack/arc.js)
+
+# DAOstack Arc.js
+
+[DAOstack Arc.js](https://github.com/daostack/arc.js) sits just above [DAOstack Arc](https://github.com/daostack/arc) on the DAO stack.  It is a library that facilitates javascript application access to the DAOstack Arc ethereum smart contracts.
+
+For more information about Arc contracts and the entire DAOstack ecosystem, please refer to the [Arc documentation](https://daostack.github.io/arc/README/).
+
+
+### Documentation
+Check out the [complete documentation on Arc.js](https://daostack.github.io/arc.js).  **Please note** that viewing documents directly in the Arc.js GitHub repository is not supported as a means of obtaining Arc.js documentation online, you will find that certain document functionality does not work when viewed that way.
```

Comment:
Why?

Lets do that same as ARC.

Reply:
> Why?

Assuming you suggesting that we explain why, so I added an explanation.

Reply:
I mean that same as in ARC all links should be valid within the repo from GitHub.com